src/main/java/com/sprint/mission/discodeit/entity/User.java

```diff
@@ -28,24 +31,24 @@ public class User extends BaseUpdatableEntity {
     @OnDelete(action = OnDeleteAction.SET_NULL)
     private BinaryContent profile;
 
-    @OneToOne(mappedBy = "user")
-    private UserStatus status;
+    @Column(nullable = false)
+    Role role;
```

Comment:
`@Enumerated(EnumType.STRING)` 애노테이션 안붙이면 JPA가 테이블 변경 시 `smallint` 타입으로 컬럼을 생성합니다!

```suggestion
    @Column(nullable = false)
    @Enumerated(EnumType.STRING)
    Role role;
```